### PR TITLE
Create dependabot.yml

### PR DESCRIPTION
**Pull request by FaisalAfroz**

### CODE REVIEW

**Comment by ynfle** [comment]
What do we depend on?

**Comment by FaisalAfroz** [comment]
actins/checkout and micnncim/action-label-syncer in workflows

File: .github/dependabot.yml
Comment (ynfle) [inline .github/dependabot.yml:11]: It seems there is still an ongoing conversation about this in https://github.com/exercism/org-wide-files/issues/92.

File: .github/dependabot.yml
Comment (ynfle) [inline .github/dependabot.yml:6]: In the [nim-test-runner](https://github.com/exercism/nim-test-runner/blob/5c1a070ca424a77fd8923aba667428438e30c0f0/.github/dependabot.yml#L11) there is also a `docker` ecosystem for dependabot PRs. Can you open an issue to look in to this or add it in here if you think it's useful?

**Review by ynfle — CHANGES_REQUESTED** [review]
Thanks for your contribution.
It seems the label discussion is still ongoing, so, if you want to merge this now, I think it's best to remove the `labels` key unless it has already been decided that it is the best route forward.

File: .github/dependabot.yml
Comment (FaisalAfroz) [reply .github/dependabot.yml:6]: Docker won't be useful here as there is no dockerfile in this repo. You can always update this file when a Dockerfile is added . For now this is sufficient.

File: .github/dependabot.yml
Comment (FaisalAfroz) [reply .github/dependabot.yml:11]: Thanks for the discussion link. I think this could be overridden org-wide later on. So it depends on the maintainers to remove it or not.

Comment (ynfle) [reply .github/dependabot.yml:11]: I think I'm gonna wait for @ErikSchierboom 's input tomorrow and then I'll merge. Once again, thanks for doing this

Comment (ErikSchierboom) [reply .github/dependabot.yml:11]: I think for now, I'd probably remove the label until we've come to a decision in the org-wide-files.

Comment (ynfle) [reply .github/dependabot.yml:11]:
```suggestion
```
@FaisalAfroz please make the requested changes and then we'll merge

**Review by ynfle — APPROVED** [review]
Thanks for the PR!

**Comment by ynfle** [comment]
We need to wait for someone from @exercism/maintainers-admin

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**
